src/tiledb/cloud/bioimg/types.py

```diff
@@ -0,0 +1,8 @@
+import enum
+
+
+class SupportedExtensions(enum.Enum):
+    TIFF: str = ".tiff"
+    TIF: str = ".tif"
+    SVS: str = ".svs"
+    TDB: str = ".tdb"
```

Comment:
This seems like it could be a module-level variable in helpers.py, just as a tuple.

Reply:
Indeed. However in a next PR I am introducing also another type and I thought it would better organised this way. I can change of course.

Reply:
I would say using an enum is useful for situations where you want to provide a configuration parameter, but it looks like in this case you’re just using it to list out suffixes. In that case, I think just using a tuple like `_SUPPORTED_EXTENSIONS = (".tiff", ".tif", ".svs", ".tdb")` makes more sense.